lib/v2/gov/sichuan/deyang/govpublicinfo.js

```diff
@@ -0,0 +1,116 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+const timezone = require('@/utils/timezone');
+
+// 各地区url信息
+const basicInfoDict = {
+    绵竹市: {
+        rootUrl: 'https://www.mz.gov.cn',
+        infoType: {
+            fdzdnr: {
+                basicUrl: 'https://www.mz.gov.cn/info/iList.jsp?node_id=GKmzs&cat_id=15971&cur_page=',
+                name: '法定主动内容',
+            },
+            gsgg: {
+                basicUrl: 'https://www.mz.gov.cn/info/iList.jsp?node_id=GKmzs&cat_id=24186&cur_page=',
+                name: '公示公告',
+            },
+        },
+    },
+};
+
+const getInfoUrlList = async (rootUrl, infoBasicUrl) => {
+    const infoUrlList = [];
+    const expectToday = new Date();
+    let notTodayCount = 0;
+    let pageNum = 1;
+    while (pageNum) {
+        // eslint-disable-next-line no-await-in-loop
+        const response = await got(`${infoBasicUrl}${pageNum}`);
+        const $ = cheerio.load(response.data);
+        // 非当前日期文章计数，部分旧文章可能会置顶，目前为发现置顶数超过10
+        const infoList = $('body > div.container > div.ewb-white > div.ewb-job > ul > li');
+        // 判断当前页是否有数据
+        if (infoList.length) {
+            for (const infoItem of infoList) {
+                const date = parseDate(timezone($('span', infoItem).html()), +8);
+                // 判断当前信息日期是否是今日
+                if (date.getUTCFullYear() === expectToday.getUTCFullYear() && date.getUTCMonth() === expectToday.getUTCMonth() && date.getUTCDate() === expectToday.getUTCDate()) {
+                    infoUrlList.push({
+                        title: $('a', infoItem).attr('title'),
+                        url: `${rootUrl}${$('a', infoItem).attr('href')}`,
+                    });
+                } else {
```

Comment:
But why push the entry only if it's today?
Users won't get any items on 2/4 if all items are from 1/4.

Reply:
I know this problem.Always, today's news are needed.
If has no limit,I don't hava a better way to handle so many news when first request this router with no limit params,and in this case, caches will become so big.
can you have some  advise, i can change this.
thx.

Reply:
I think scraping only the first page is fine IMO. It will only send 16 requests on the first RSS generation and should occupy ~4MB of cache.

Reply:
I have modified it as you said.
Thx.